app/src/main/java/com/github/multimatum_team/multimatum/viewmodel/GroupViewModel.kt

```diff
@@ -69,6 +69,12 @@ class GroupViewModel @Inject constructor(
     fun getGroups(): LiveData<Map<GroupID, UserGroup>> =
         _groups
 
+    /**
+     * Get all owned groups.
+     */
+    fun getOwnedGroups(): Map<GroupID, UserGroup> =
+        _groups.value!!.filterValues { group -> (group.owner == authRepository.getUser().id) }
```

Comment:
I am sorry to insist that much on comments, but the Javadoc of this function is not really useful. Having a Javadoc here can be a good idea, of course, but it is useful only if it says something more than the name of the method.

Reply:
I have just copied what was done in the other function of the class